### ISSUE TITLE
News related ADA issues

### ISSUE THREAD

**Issue by bbusenius** [uchicago-library/library_website#663]
## LIB20230113-16. Alt text on image is being overridden (Major)

**Issue**: The good alt text on the image inside the article is being overridden by a bad aria-label on the wrapping <a> tag. The aria-label should be removed.
```
<a href="/media/images/Torsten_Reimer_110-ns.original.jpg" data-toggle="lightbox" aria-label="larger view of Torsten_Reimer_110-ns.jpg">
	<img src="/media/images/Torsten_Reimer_110-ns.width-800.jpg" 
	alt="Torsten Reimer in front of Regenstein Library framed by 
	Hull Gate" class="img-responsive">
</a>
```
[Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit#heading=h.rjpdeyv91r6k)

## LIB20230113-17. Category badge links markup (Moderate)

**Issue**: The category badge links below the story are marked up in series without any separation, so screen readers have some difficulty accurately announcing these elements. Consider marking these up in an unordered list (maintaining the existing visual style) and including a visible or a visually hidden (screen reader only) `<h2>` above. For example:
```
<h2 class="sr-only">Categories</h2>
<ul>
	<li><a href="..." class="badge">From the Dean</a></li>
	<li><a href="..." class="badge">Research</a></li>
	<li><a href="..." class="badge">Spaces</a></li>
	<li><a href="..." class="badge">People</a></li>
</ul>
```
[Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit#heading=h.2psg2qvhygmi)

## LIB20230113-10. News section article block markup (Moderate)

**Issue**: The article block markup could be simplified/improved from a semantic HTML standpoint.
The image and category shouldn’t be included in figure and figcaption tags since the images aren’t figures and the categories aren’t captions for the images.
The alt text on the images could be improved,
The headline `<h3>`s should include the `<a>` as children, not the other way around.
The repeated "Read More..." link text is too vague and not unique, so these links require unique aria-label attributes.

For example:
```
<div class="newsblock col-xs-12 col-sm-6 col-md-3">
	<div class="figure-wrap">
		<div class="news-category exhibit-news">Exhibit
		News</div>

		<img alt="Late 19th-century toy book, Cinderella, from 
		the exhibition But is it a Book" class="img-responsive" 
		height="125” src="..." width="273">
	</div>

	<h3><a href="...">But Is It a Book?</a></h3>

	<p>
		Opening Jan. 3, this exhibition explores the nature of 
		the material text, from the clay tablet to the
		contemporary artist’s book.<br>
		<a href="..." aria-label="Read more about But Is It a 
		Book?">Read more...</a>
	</p>
</div>
```
[Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit#heading=h.5qfm1nbb5lx8)

## LIB20230113-18. More News section article block markup (Moderate)

**Issue**: The More News section is similar to the News Section on the Homepage issue above, but the markup is slightly different. We suggest mirroring the recommended markup from the homepage issue above, but omitting the blurb and “Read more...” links. For example:
```
<div class="newsblock col-xs-12 col-sm-6 col-md-3">
	<div class="figure-wrap">
		<span class="news-category law">Law</span>
		<img alt="Malcolm X" class="article-img" height="200" 
		src="..." width="358">
	</div>

	<h3><a href="...">January Resource of the Month - Federal 
	Surveillance of African-Americans, 1920-1984</a></h3>
</div>
```
[Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit#heading=h.y9941b40oqlo)
